config/settings-env/endpoints.json

```diff
@@ -550,6 +550,20 @@
         "timeout": "600s",
         "input_query_strings": []
       },
+      {
+        "endpoint": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "url_pattern": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "method": "POST",
+        "timeout": "600s",
+        "input_query_strings": []
+      },
+      {
+        "endpoint": "/v1beta/sse",
```

Comment:
it is unclear to what extend krakend actually supports SSE, probably have to develop a plugin for it